spotbugs/src/main/java/edu/umd/cs/findbugs/TextUICommandLine.java

```diff
@@ -380,22 +403,33 @@ protected void handleOption(String option, String optionExtraPart) {
             trainingInputDir = !"".equals(optionExtraPart) ? optionExtraPart : ".";
         } else if ("-html".equals(option)) {
             bugReporterType = HTML_REPORTER;
-            HTMLBugReporter htmlBugReporter = new HTMLBugReporter(project, "default.xsl");
-            optionExtraPart = handleOutputFilePath(htmlBugReporter, optionExtraPart);
-            if (!"".equals(optionExtraPart)) {
-                htmlBugReporter.setStylesheet(optionExtraPart);
+            TextUIBugReporter reporter = initializeReporter(optionExtraPart,
+                    () -> new HTMLBugReporter(project, "default.xsl"),
+                    (r, rOpt) -> {
+                        if (!"".equals(rOpt))
+                            r.setStylesheet(rOpt);
```

Comment:
Can you please add the brackets to the if to be conform to the Google style guide?